src/components/Form/formActions.ts

```diff
@@ -181,13 +176,16 @@ export const formActions = {
 					})
 				);
 			}
-
+		}
+	},
+	disableForm(disabled: boolean) {

```

Comment:
Let's keep the args as object to maintain the same "structure" as the other actions.
```suggestion
	disableForm({disabled = false}: {disabled: boolean}) {

```